wizeline/project-lab
# Feature/count of projects#184

#### What does this PR do?

Add count total of projects in search section



#### Where should the reviewer start?

- app/pages/projects/search.tsx



#### How should this be manually tested?

To be able to see the changes, the user needs to be a member of a Project.

On the search section the user should see the total of projects that belongs to the category selected



#### Any background context you want to provide?



#### What are the relevant tickets?

[Show count of projects in current search](https://github.com/wizeline/project-lab/issues/184)



#### Screenshots
![Screen Shot 2022-02-14 at 16 10 28](https://user-images.githubusercontent.com/98923371/153955327-06d0c429-f019-4962-a9d2-a604d16ec94e.png)





#### Questions



#### Checklist
